src/components/DataSection/DataSection.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import './DataSection.scss';
+import { getLocation } from '../../commons/helpers';
+import Card from '../Card/Card';
+import { Tabs } from '../../statics/Statics';
+
+function DataSection ({activeTab}) {
+    const [data, setData] = useState([]);
+    const [loading, setLoading] = useState(false);
+
+    useEffect(() => getData(), [activeTab])
```

Comment:
You return a cleanup function to useEffect. You should not return anything if you do not want to perform a cleanup to avoid confusion. https://reactjs.org/docs/hooks-effect.html